plugins/system/jooa11y/jooa11y.php

```diff
@@ -260,14 +234,46 @@ public function initJooa11y()
 			Text::script($constant);
 		}
 
-		/** @var Joomla\CMS\WebAsset\WebAssetManager $wa*/
+		/** @var Joomla\CMS\WebAsset\WebAssetManager $wa */
 		$wa = $document->getWebAssetManager();
 
 		$wa->getRegistry()->addRegistryFile('media/plg_system_jooa11y/joomla.asset.json');
 
 		$wa->useScript('plg_system_jooa11y.jooa11y')
 			->useStyle('plg_system_jooa11y.jooa11y');
+	}
+
+	/**
+	 * Method to check if the current user is allowed to see the debug information or not.
```

Comment:
```suggestion
	 * Method to check if the current user is allowed to see the checker or not.
```

I know this was an existing bug but might as well get fixed here at the same time